PowerPointLabs/PowerPointLabs/Utils/Graphics.cs

```diff
@@ -457,7 +466,16 @@ public static void MoveZUntilInFront(Shape shiftShape, Shape destinationShape)
         /// </summary>
         public static void MoveZUntilBehind(Shape shiftShape, Shape destinationShape)
         {
-            while (shiftShape.ZOrderPosition > destinationShape.ZOrderPosition)
+            MoveZUntilBehind(shiftShape, destinationShape.ZOrderPosition);
+        }
+
+        /// <summary>
+        /// Moves shiftShape backward until it is behind destination zOrder index.
+        /// (does nothing if already behind)
+        /// </summary>
+        public static void MoveZUntilBehind(Shape shiftShape, int destination)
+        {
+            while (shiftShape.ZOrderPosition > destination)

```

Comment:
Seems like this is not working as intended, since the parameter is a constant instead of a reference when it is changing.